src/aleph/netty.clj

```diff
@@ -1516,6 +1518,14 @@
                     (ssl-handler ch ssl-ctx))))
      (pipeline-builder p))))
 
+(def ^:const
+  default-connect-timeout
```

Comment:
I would just put the scalar (duplicate) with a comment that points to the class definition.
That's not the first time we have done this : https://github.com/clj-commons/aleph/blob/master/src/aleph/netty.clj#L192-L194

Reply:
Good point, [did that](https://github.com/clj-commons/aleph/pull/715/commits/8ace007d5f0ab24b75838f2ee87178f8335acb8b). I also moved it to the existing section for this but unlike `default-shutdown-timeout`, I opted to *not* mark it as `:no-doc` so that we can still refer to it in the docstrings. I guess this would make sense for `default-shutdown-timeout`, too. WDYT?

Reply:
Since Aleph is not strongly encapsulating its internals, I tend to think : "If it's not documented, it's not supposed to be used".
If we remove the `:no-doc`,  we basically mention its part of the Aleph API.

I would say we can remove both the `no-doc`and we can commit on it.

Reply:
Makes sense, pushed a commit to that effect :+1: